src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1510,6 +1512,19 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     }
   }
 
+  def getBucketUsage(workspaceName: WorkspaceName): Future[PerRequestMessage] = {
+    for {
+      bucketName <- dataSource.inTransaction { dataAccess =>
+        withWorkspaceContextAndPermissions(workspaceName, WorkspaceAccessLevels.Read, dataAccess) { workspaceContext =>
+          DBIO.successful(workspaceContext.workspace.bucketName)
+        }
+      }
+      usage <- gcsDAO.getBucketUsage(RawlsBillingProjectName(workspaceName.namespace), bucketName)
+    } yield {
+      RequestComplete(BucketUsageResponse(usage))
+    }
+  }
```

Comment:
There are 2 cases I can think of when this might happen:

1. The workspace and bucket have just been created and therefore have no logs yet.
2. There's a bug in rawls or someone has manually interfered with the logging config or log objects.

For case #1, a 204 might be okay, as would a 200 with a result of 0.
For case #2, I think a 500 is entirely appropriate. 2xx doesn't seem right because that would indicate success. 4xx doesn't seem right because there isn't anything wrong with the client request. I didn't see any better 5xx codes, so I still think 500 covers this.

The trick then is being able to distinguish between #1 and #2. Since there are potentially no storage log files in both cases, my only thought is to immediately write a fake storage log when a workspace is created. Then, the lack of any log objects represents an unexpected server issue and, therefore, a 500 response.